modules/dataquery/help/DQThelp.md

```diff
@@ -0,0 +1,47 @@
+# Data Query Tool
```

Comment:
If you want to change the content of DQT help dropdown. You need to modify the dataquery.md directly, do not create a new one file.

Reply:
good catch, @kongtiaowang Shen! 👍

Reply:
Hi @kongtiaowang @christinerogers I can't rename the file because there is already an existing file called dataquery.md (which has been removed in PR #5206, specifically [this commit](https://github.com/aces/Loris/pull/5206/commits/b1500120436f4464210fe7cb879c868d9fe0458c) - however it still exists because that PR hasn't been merged yet.

Reply:
Hi Alex - I see what you mean, you'll likely have to re-base one of these PRs - that's ok, that's a step on its own.  
For now, try renaming this file and let's see if we can get it merged first -- since the Module Spec might need more time for content review.

Reply:
Thanks - this should be fixed now.